docs/conf.py

```diff
@@ -82,6 +82,27 @@
           'the "glymur" package to be installed')
     sys.exit(1)
 
+try:
+    import sqlalchemy
+except ImportError:
+    print('ERROR: sqlalchemy could not be imported. Building the documentation requires '
```

Comment:
Rather than using `print` and `sys.exit`, I think it'd be better to catch and reraise the `ImportError` with a custom message, e.g.

```python
try:
    import sqlalchemy
except ImportError as e:
    raise Exception(e, 'ERROR: sqlalchemy could not be imported. ...')
```

Reply:
Sold.